server/channels/app/team.go

```diff
@@ -1391,8 +1391,13 @@ func (a *App) InviteNewUsersToTeamGracefully(rctx request.CTX, memberInvite *mod
 			Email: email,
 			Error: nil,
 		}
+
+		userToInvite, _ := a.GetUserByEmail(email)
```

Comment:
Should we handle the error a bit better here? If it's not a 404, we should log an error and continue.

Reply:
How do I check if the error is a 404?

Reply:
Something like this?
```suggestion
		userToInvite, err := a.GetUserByEmail(email)
		if err.StatusCode == http.StatusNotFound {
			invite.Error = model.NewAppError("InviteNewUsersToTeam", "api.team.invite_members.invalid_email.app_error", map[string]any{"Addresses": email}, "", http.StatusBadRequest)
		} else if err != nil {
			invite.Error = model.NewAppError("InviteNewUsersToTeam", "app.user.get.app_error", nil, "", http.StatusInternalServerError)
		}

```

Reply:
Looks good :+1: